helm/vitess/templates/_vttablet.tpl

```diff
@@ -83,7 +84,7 @@ spec:
         shard: {{ $shardClean | quote }}
         type: {{ $tablet.type | quote }}
     spec:
-      terminationGracePeriodSeconds: 600
+      terminationGracePeriodSeconds: 30
```

Comment:
Why the revert? Was this causing problems?

Reply:
That was an accidental commit. I turned it down while testing it out so my `helm del` finished faster.